portfolio/src/main/webapp/template.html

```diff
@@ -65,56 +82,88 @@ <h2 class="heading">About Me</h2>
                         <h2 class="heading">Latest Projects</h2>
                         <!--TODO: add in microcontroller engineering project -->
                         <div class="content">    
-                                               
                             <div class="item featured text-center">
-                                
                                 <div class="featured-image has-ribbon">
                                     <a href="#" target="_blank">
-                                    <img class="img-fluid project-image rounded shadow-sm" src="assets/images/projects/GradTracks.png" alt="GradTracks" />
+                                    <img    
+                                        class="img-fluid project-image rounded shadow-sm" 
+                                        src="assets/images/projects/GradTracks.png" 
+                                        alt="GradTracks" />
                                     </a>
                                     <div class="ribbon">
                                         <div class="text">New</div>
                                     </div>
                                 </div>
-                                
-                                <h3 class="title mb-3"><a href="#" target="_blank">GradTracks - The best way to visualize alumni paths and outcomes</a></h3>
-                                    
+                                <h3 class="title mb-3">
+                                    <a href="#" target="_blank">GradTracks - The best way to visualize alumni paths and outcomes</a>
+                                </h3>                                    
                                 <div class="desc text-left">                                    
-                                    <p>Lead developer for databases class group project. GradTracks is a tool that allows students to see what previous alumni have done after graduating. Built database, backend and frontend that quickly processed queries and displayed data visualizations.</p>
+                                    <p>Lead developer for databases class group project. GradTracks is a tool that allows students to 
```

Comment:
Nit: I like putting text on the next line + indent one level in these situations.

Reply:
@g-harel can you clarify?

Reply:
Yeah sorry, should have done a diff to start. This is how I like to format multiline/long text content:

```diff
- <p>lorem ipsum
+ <p>
+    lorem ipsum
     lorem ipsum
  </p>
```

Reply:
Thank you @g-harel  - just implemented!